Application/EdFi.Ods.Common/Models/Resource/ResourceClassBase.cs

```diff
@@ -269,7 +269,7 @@ public string PluralName
         /// <summary>
         /// Gets the identifying properties of the resource that are introduced in the local context, including those in references, as necessary.
         /// </summary>
-        public IReadOnlyList<ResourceProperty> IdentifyingProperties => AllProperties
+        public virtual IReadOnlyList<ResourceProperty> IdentifyingProperties => AllProperties
```

Comment:
These members were made virtual to support unit tests which were too difficult to setup without using mocks.